### ISSUE TITLE
Using Jsmooth from Griffon?

### ISSUE THREAD

**Issue by seanleblanc** [griffon/griffon#249]
I'm thinking that this is out of date? https://github.com/griffon/griffon-installer-plugin

If that's the case, is it possible to get a native windows installer (like what Jsmooth offers) to work from Griffon?

**Comment by aalmiray** [comment]
That's correct. The `griffon-installer-plugin` only works with Griffon 1.x.
For Griffon 2.X you would have to use a plugin for the target build system of your choice (Gradle or Maven). Projects created with the default templates include basic setup for uberjar (via shadow or shade), zip distribution, and IzPack.

If the application is JavaFX based then you also have access to the javapackager with default configuration. Lastly, on Gradle builds the `macappbundle` plugins is also configured by default. You could also try the `oss-package` plugin from Netflix' Nebula in case you need to target RPM, Deb, or other popular Linux based formats.

If there's a JSmooth plugin for Maven and/or Gradle then it's just a matter of configuring in your build.
